scene/gui/split_container.cpp

```diff
@@ -101,30 +101,32 @@ void SplitContainerDragger::_notification(int p_what) {
 				queue_redraw();
 			}
 		} break;
-
 		case NOTIFICATION_MOUSE_EXIT: {
 			mouse_inside = false;
 			SplitContainer *sc = Object::cast_to<SplitContainer>(get_parent());
 			if (sc->theme_cache.autohide) {
 				queue_redraw();
 			}
 		} break;
-
 		case NOTIFICATION_DRAW: {
 			SplitContainer *sc = Object::cast_to<SplitContainer>(get_parent());
-			if (!dragging && !mouse_inside && sc->theme_cache.autohide) {
-				return;
+			draw_style_box(sc->theme_cache.split_bar_background, split_bar_rect);
+			if (!(sc->dragger_visibility != sc->DRAGGER_VISIBLE || (!dragging && !mouse_inside && sc->theme_cache.autohide))) {
```

Comment:
```suggestion
			if (sc->dragger_visibility == sc->DRAGGER_VISIBLE && (dragging || mouse_inside || !sc->theme_cache.autohide)) {
```

Reply:
Busted with my hand in the cookie jar of laziness... 🍪

Reply:
Done.